Fix cliplines bug

MattiasBeming opened the pull request:
All cliplines are now shown - scroll down to view all cliplines. 
When changing scale, scroll timemarker into view. 
added more scaling options for timeline.
removed console.log in map.js

Closes #396

MattiasBeming committed:
fix: scroll to see all cliplines

Scroll timemarker into view when changing scale

MattiasBeming committed:
feat: add more scaling options, set direction in dropdown menus to go up

MattiasBeming committed:
fix: remove console.log in map.js

sporreking reviewed (changes requested):
Timemarker does not align with clips due to the decreased inner width in _cliplines_ caused by scrollbar.

MattiasBeming committed:
fix: make scrollbar invisible

Fixes the bug where Timemarker does not align with clips due to the decreased inner width in cliplines caused by scrollbar.
But it does make it unclear to the user that it is possible to scroll vertically in player-mode, to see all clips.

sporreking reviewed (approved):
Looks good!
